quesma/quesma/config/config_v2.go

```diff
@@ -573,6 +561,29 @@ func (c *QuesmaNewConfiguration) TranslateToLegacyConfig() QuesmaConfiguration {
 	}
 
 END:
+
+	if err != nil && !conf.TransparentProxy {
```

Comment:
Nit: since the distance between setting `err` on line 472 (`relDBConn, connType, err := c.getRelationalDBConf()`) and this actual use of `err` is so large, maybe it'd be better to rename `err` to `relationalDBErr`.

Reply:
good call https://github.com/QuesmaOrg/quesma/pull/816/commits/020b45d1bd59ae6caabe51ffee56f1a463a0af80